profile/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.7"
+
+services:
+  ldaptive:
+    image: maven:3.6-jdk-11
+    ports:
+      - 9003:9003
+      - 10001:10001
+    volumes:
+      - $HOME/.m2:/root/.m2
+      - $PWD:/apps/ldaptive
+    environment:
+      - HOST=ldap-test
+      - PORT=389
+      - BASE_DN=ou=test,dc=vt,dc=edu
+      - BIND_DN=uid=1,ou=test,dc=vt,dc=edu
+      - BIND_CREDENTIAL=VKSxXwlU7YssGl1foLMH2mGMWkifbODb1djfJ4t2
+      - PROFILE_CLASS
+      - THREAD_COUNT
+      - THREAD_SLEEP
+      - ITERATIONS
+      - USE_YOURKIT
+    networks:
+      ldap_net:
+        ipv4_address: 172.28.1.10
+    command: >
```

Comment:
I don't see any mechanism to make your client wait on directory startup/initialization.

Reply:
That's a good point. No wait exists for the integration tests either. Openldap always starts in time....

Reply:
I think that's a fine answer for test fixtures, but if integrating `wait-for-it.sh` or similar isn't too difficult, then maybe it makes your tools a little more durable.

Reply:
I think this works because in all my use cases a maven compile happens before tests that depend on Openldap are executed. That gives plenty of time for the Openldap container to start.

Reply:
Makes sense.